draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -795,6 +795,16 @@ This document has no actions for IANA.
 </section>
 </section>
 
+<section title="Prefix-Specific Link-Local Addresses">
+<t>
+Various router vendors already allow administrators to configure multiple link-local addresses on a given interface (e.g. configuring an interface link-local and multiple VRRP link-local addresses).
```

Comment:
What are the implementations? And... what are the rules they use when sending packets? (i.e., which LLA do they use for what?)

Reply:
My observations that there was no rules (which is not good) - I observed RAs from both VRRP link-local source and from configured "real" link-local.